app/components/playfair/Recreation.tsx

```diff
@@ -0,0 +1,259 @@
+  const pathImport = "M0.245085 172.502C9.15446 172.371 16.5835 172.075 22.0447 171.81C31.3302 171.347 36.0547 171.099 40.1282 170.277C47.6018 168.787 48.6682 167.418 56.2578 166.461C59.9197 166.17 63.5612 165.658 67.1636 164.928C70.7434 164.134 71.3387 163.696 74.854 162.993C79.0337 162.158 79.6852 162.473 85.0811 161.743C89.6278 161.133 90.0427 160.8 97.2439 159.559C100.512 158.994 103.549 158.478 106.774 158.075C114.78 157.092 116.531 157.784 127.625 157.158C134.923 156.737 134.059 156.45 140.432 156.172C149.612 155.763 150.315 156.423 156.767 155.934C164.883 155.315 164.33 154.232 172.773 153.522C181.476 152.783 181.664 153.978 189.357 152.927C194.27 152.255 195.237 151.624 201.949 150.807C206.305 150.274 207.22 150.38 211.156 149.772C217.445 148.801 217.23 148.213 221.433 147.745C228.128 146.998 229.305 148.455 233.652 147.128C236.326 146.32 238.096 145.029 241.421 142.603M241.613 142.463C249.238 136.816 248.805 134.756 253.558 131.785C256.916 129.679 258.343 129.985 269.138 127.254C277.646 125.131 279.743 125.453 283.046 123.428C286.547 121.199 289.391 118.039 291.274 114.284C291.301 113.574 291.587 112.901 292.075 112.397C292.563 111.892 293.218 111.593 293.911 111.558C296.475 111.377 296.663 114.18 296.663 114.18C296.663 114.18 300.861 168.204 301.271 173.423C301.78 174.497 302.58 175.399 303.576 176.02C304.572 176.642 305.722 176.956 306.887 176.925C311.395 176.7 315.713 175.011 319.217 172.104C321.209 170.046 322.055 168.218 324.043 163.598C328.994 152.089 332.329 143.936 332.329 143.936C336.19 134.482 337.145 132.451 339.541 130.27C341.246 128.937 342.772 127.38 344.079 125.638C346.193 122.653 346.103 120.968 347.549 117.46C348.359 115.444 351.028 108.983 356.806 105.121C359.43 103.358 359.919 104.016 363.872 101.451C367.294 99.233 367.401 98.4347 370.496 96.8241C372.475 95.76 373.42 95.597 376.55 93.8957C378.709 92.7159 379.693 92.0153 380.467 90.8653C381.22 89.6642 381.643 88.2767 381.69 86.849C382.016 82.418 382.278 87.2748 383.795 74.1511C384.67 66.6218 384.711 63.9688 386.642 60.2052C387.055 59.4011 387.512 58.622 388.013 57.8718M241.613 142.463L241.421 142.603M241.613 142.463C241.548 142.51 241.484 142.556 241.421 142.603";
+
+  const ellipse = { cx: 28, cy: 17, rx: (94 / 11) * 1.9, ry: 10 };
+  const topText = { text: "EXPORTS & IMPORTS", x: 14, y: 15 };
+  const midText = { text: "to and from all", x: 22, y: 18.5 };
+  const botText = { text: "NORTH AMERICA", x: 15, y: 22 };
+
+
+  const formatYValue = (value) => {
```

Comment:
We should probably put this outside the component as it does not depend on the component props.